### PR TITLE
do not run yarn berry with `--immutable`

### PR DESCRIPTION
**Pull request by coffee-cup**
## What does this PR address?
Yarn berry was seemingly updating the lockfile even after it was just generated. Not exactly sure why it is happening but since it can happen, we probably shouldn't be installing the deps with `--immutable`.

### CODE REVIEW

**Comment by Milo123459** [comment]
Do you mean we probably should**n't** be installing deps with --imutable?

**Comment by coffee-cup** [comment]
Oh yeah, typo. Updated the original

**Review by Milo123459 — DISMISSED** [review]
LGTM

**Review by AhmedMozaly — APPROVED** [review]

File: tests/docker_run_tests.rs [hunk 1]
```diff
@@ -253,6 +253,9 @@ fn test_yarn_custom_version() {
 fn test_yarn_berry() {
     let name = simple_build("./examples/node-yarn-berry");
     let output = run_image(name, None);
+
+    println!("Output: {}", output);
```
Comment (AhmedMozaly) [inline tests/docker_run_tests.rs:257]: Why we need to log output?

Comment (coffee-cup) [reply tests/docker_run_tests.rs:257]: We do not, I was just testing stuff. Good catch

**Review by AhmedMozaly — APPROVED** [review]

**Review by Milo123459 — APPROVED** [review]

**Pull request merged**